packages/runtime/src/main/resources/META-INF/native-image/reflect-config.json

```diff
@@ -2438,6 +2438,118 @@
   "name":"java.util.Arrays.ArrayList",
   "methods":[{"name":"toString","parameterTypes":[] }]
 },
+{
+  "name": "tools.elide.vfs.Directory",
+  "allDeclaredMethods":true,
+  "allPublicMethods":true,
+  "allDeclaredConstructors":true,
+  "allPublicConstructors":true
+},
```

Comment:
Some missing protobuf-related reflection metadata; only included because it is needed in verbose logging modes to print protos.